package.json

```diff
@@ -81,6 +81,7 @@
   },
   "scripts": {
     "test": "mocha --recursive && make test",
+    "webpack": "node node_modules/webpack/bin/webpack.js",
```

Comment:
Does windows ship with npx with npm now? Could you try running `npx webpack` i have a half finished patch that will move us towards using npx so be good to know if it works on windows :)

Reply:
what the heck is npx O_o I'm only just getting used to yarn, and I like yarn because I imagine kittens and balls of wool yarn when thinking about yarn

Reply:
" npx webpack" works

Reply:
npx is a node/npm feature that executes scripts inside node_modules. It basically ensures that `"node node_modules/webpack/bin/webpack.js"` will always run for us. It looks up webpacks path from `package.json`and executes webpack in the bin directory.

Reply:
works without the package.json change, what magic

Reply:
:D